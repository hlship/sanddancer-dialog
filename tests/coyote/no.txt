(grab [#pack #strength])
(setup coyotes offer)
no
advice
ask who
rabbit
trading
luck
scent
trade stories for scent
trade job for scent
i
n
