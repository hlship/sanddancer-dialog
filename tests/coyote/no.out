

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (grab [#pack #strength])
Query succeeded: (grab [#pack #strength])
> (setup coyotes offer)

Open Desert
Your tracks are swallowed up in moments by the billowing sand, and you shiver in
the chill night air.

All you can make out in the darkness are a looming cactus, a stand of cactus,
and some dry bones.

And then you blink, and realize he's not a coyote after all. Just... just a guy.

He wears a couple of days of stubble, a scuffed and faded denim jacket, and,
ridiculously in the darkness, a pair of cheap sunglasses. He lights a cigarette,
then holds up a hand in annoyance to block your flashlight beam. "Turn that
thing off," he says, and either you do it or it goes off by itself, because the
next thing you know, it's darker and he's closer and the only light comes from
the red glow of the cigarette.

"That's better," he says. He takes a drag on the cigarette, then pulls another
from behind his ear and offers it to you. "Smoke?"

You could say yes or say no.
Query succeeded: (setup coyotes offer)
> no
"Nah, I'm trying to kick 'em" you say.

He shugs, takes a puff on his, and exhales the smoke into the desert night. Wind
whips it away and he stares towards the horizon, forehead wrinkling in something
(annoyance, maybe contemplation) you can't read through the sunglasses.

The coyote stares at you momentarily before taking a drag of his cigarette.
"Gotta keep an eye on the horizon," he says in a scratchy, slow voice. "Always
someone hunting. Border Patrol, National Guard, INS, ICE, Minutemen. You want to
help people get where they're going out here, you need more than strength. You
need to learn how to hide."

"I'm the Coyote," he says, "And we've been hiding from each other for a long
time."

You could ask for advice, ask who he is, ask about the rabbit, or ask about
trading.

> advice
"Ok, sure," you say, "I'll take some advice if you got it."

He laughs, itching underneath his jacket, slowly. "Kid," he says, "you don't
want my advice. I could tell you some stories and you'll smile and be sure
they'll never happen to you. I could tell you some stories and you'll yawn and
think you know them already. I could tell you some stories and you'll think
they're wise and not realize you don't understand them until you make the same
mistakes yourself and try to turn your failure into advice and fail at that,
too. I could tell you some stories." He looks off towards the horizon, the red
tip of the cigarette glowing as he takes another drag. "But it's not my stories
that will help you."

You could ask who he is, ask about the rabbit, or ask about trading.

> ask who
"No, really, who are you?" you ask.

"You know who I am," he says, smoke gusting from his lungs in clouds. "I'm the
one who runs on both sides of the fence. I'm the one who scouts ahead. I'm the
one with songs in my blood and dirt under my nails and people owe me money in
every trailer park town from here to Yuma, or maybe I owe them money but anyway
they don't want to see me. You know who I am, Knock."

You could ask about the rabbit or ask about trading.

> rabbit
"Are you like the rabbit?" you ask.

He grins, not unkindly but not exactly kindly either. "That cute little
jackalope?" he says, the red tip of the cigarette glowing as he takes another
drag. "He's got his upsides, I guess. Strength has its place. But if you're
always in the spotlight, you'll never learn how to live in the shadows. And most
of us spend a lot of time in the shadows."

You could ask about trading.

> trading
You nervously ask, "Rabbit did a trade. What about you?"

The Coyote presses his hands together, still clutching the cigarette with thumb
and index finger, and smoke rises past his hidden eyes as he slowly rubs his
palms together. He mutters something rhythmic to himself and you realize it's a
song, and you can't quite make out the lyrics, but the melody is simple and
makes you think of Johnny Cash and the end of the world. And then he stops, and
pulls his hands apart, and in each he holds a talent.

"Here we have luck," he says, bobbing the left, "and scent. And look, I'll be
straight with you. These are precious things and I can't afford whatever blue
light special deal the bunny gave you. I need double his price. I need two
memories to part with one of these. Oh, and they're worth it, believe me. Both
have saved my ass more times than you'd believe. Both could save yours. But it's
up to you. Which'll it be?"

You could ask about luck or ask about scent.

> luck
"What's happened tonight", you say, "what's happened before tonight. Maybe I
could use some luck."

"Looks like you're already pretty tough," he says appraisingly, look you over.
"With a little luck, you ought to make it through the night and get on with the
rest of your life."

You could ask about scent.

> scent
"What's scent going to do for me?" you ask.

"You're pretty tough," he says, "but I don't know that a good nose is going to
help you make it through the night and get on with the rest of your life, if
that's what you're trying to do."

> trade stories for scent
The Coyote touches your fingertips and pulls the memory of your grandma's
stories through your nerve endings. "Good," he says, "I'll take that, although
it's not enough to complete the trade."

Somewhere inside you, things are shifting, moving, growing.

> trade job for scent
The Coyote nods. "Yes," he says, "a fair trade." And something happens inside
you as he says it. The memory of your shit job shifts and wriggles and fades
away and it's still there, but now it's shifted, become something else. And,
yeah, it kinda does feel like you could call it scent. Cool. He tucks the other
talent back in his jacket, and without its faint glow, it seems darker than
ever, the pulsing red tip of his cigarette all you can see.

"The last of us is coming," Coyote's voice says out of the black, "last and most
powerful, and most dangerous, and most afraid. Sand-dancer. Be careful of him,
Knock. Stay strong. Keep your nose to the wind. Remember the shadows."

And it seems like he's growing, larger and larger, the red tip of his cigarette
pulsing and glowing now high up in the sky, and you stumble back, shivering, and
suddenly remember your flashlight in your hand and switch it on...

Base of the Tower
Behind the concrete building a steel girder rises from the sand; far above you,
a red warning light blinks on and off. A tumbleweed drifts lazily against the
rusted metal among scrawny weeds and bits of trash. Someone left a tin can here
years ago: ridged ripples run down its rusty side.

The desert stretches in all directions except back south towards the glow of the
building.

> i
You have a flashlight, a pack of cigarettes, your lighter, your wallet, and your
emotional baggage. You're wearing your denim jacket. You've gained the talents
strength and scent.

In your emotional baggage is the memory of meeting Ocean.

Something about the tumbleweed makes you want to brood about your last day of
high school.

> n
(first attempting to switch the flashlight on)
Your surroundings spring into brightness around you.

Something about the sage makes you want to brood about watching Family Guy with
Karl.

You have a feeling that when the last spirit animal decides to make an
appearance, it will find you.

> 
